sites/all/modules/reol_app_feeds/reol_app_feeds.info

```diff
@@ -0,0 +1,8 @@
+name = eReolen app feed
+description = App feeds for eReolen and eReolen Go
+core = 7.x
+package = eReolen Base
+version = 7.x-1.0
```

Comment:
Remove the version tag as we don't use it and we don't updated it.

Reply:
Fixed.